content/en/users/Rucio/Rucio-commands/_index.md

```diff
@@ -0,0 +1,182 @@
+correctly.
+
+### upload
+
+A Rucio command that allows you to upload files from your current environment to any RSE within your VO.
+
+```shell
+$ rucio upload [-h] --rse RSE [--lifetime LIFETIME] [--scope SCOPE]
+                    [--impl IMPL] [--register-after-upload] [--summary]
+                    [--guid GUID] [--protocol PROTOCOL] [--pfn PFN]
+                    [--name NAME] [--transfer-timeout TRANSFER_TIMEOUT]
+                    [--recursive]
+                    args [args ...]
+```
+
+Several of the options you will not need to be used as they will be set by the Rucio Admins
+of your VO when they set up the RSEs.
+Below are a list of options that you may find useful:  
+
+* `RSE` is the Rucio Storage Element or site that you wish to store the data at,
+the list of available RSEs can be seen for your VO with the command
+`rucio list-rses`.
+* `Lifetime` is how long you wish the file to exist,
+not specifyting will make the file permenant until rucio is told to delete it.
+* `Scope` can be used in many ways,
+but often can be an experiment name, or a user space, all users have their own scope assigned to them `user.<username>`.
+* `Register-after-upload` allows for files to be uploaded to the destination,
+and then registered with Rucio, rather than the other way around.
+This can be useful if your connection is intermittent.
+* `Name` is the name of the file that it will be registered to Rucio with,
+if this is not set it will be the name of the file/s provided.
+* `Recursive` Allows you to set the arg to a directory,
+and all files within that directory and any sub directories will be uploaded.
+* `Args` is the path to the file, or files you wish to upload, this can be a single file,
+directory (with recursive set), or a list of files seperated with a space e.g.
+
+```shell
+rucio upload --rse main-rse file1 file2 file3 file4
+```
+
+### get
+
+A Rucio command to download files from any RSE in your VO to your local environment.
+
+```shell
+$ rucio get [-h] [--dir DIR] [--allow-tape] [--rse RSE] [--rses RSES]
+                 [--impl IMPL] [--protocol PROTOCOL] [--nrandom NRANDOM]
+                 [--ndownloader NDOWNLOADER] [--no-subdir] [--pfn PFN]
+                 [--archive-did ARCHIVE_DID] [--no-resolve-archives]
+                 [--ignore-checksum] [--transfer-timeout TRANSFER_TIMEOUT]
+                 [--transfer-speed-timeout TRANSFER_SPEED_TIMEOUT] [--aria]
+                 [--filter FILTER] [--scope SCOPE] [--metalink METALINK_FILE]
+                 [--deactivate-file-download-exceptions]
+                 [dids [dids ...]]
+```
+
+* `Dir` is the location within the container you wish for the files to be downloaded
+(if you wish to move these files outside of the container,
+you may want to mount a volume in the container to allow the files to persist).
+* `RSE/s` specifying which RSE/s you wish to download the files from,
+leaving this blank will allow Ruico to decide which RSE/s is best.
```

Comment:
```suggestion
leaving this blank will allow Rucio to decide which RSE/s is best.
```
I see here another /s, I guess it's not a typo, but it's not something we are used too, nor using elsewhere in this site. Can it be replaced by some like RSE, RSEs, or RSE(s) ?

Reply:
corrected typo and repleased the RSE/s with RSE(s)